{
  "Entries": [
    {
      "RequestUri": "https://api.loganalytics.io/v1/bad/resource/id/query",
      "RequestMethod": "POST",
      "RequestHeaders": {
        "Accept": "application/json",
        "Accept-Encoding": "gzip, deflate",
        "Content-Length": "75",
        "Content-Type": "application/json",
        "Prefer": "",
        "User-Agent": "azsdk-python-monitor-query/unknown Python/3.11.6 (Linux-5.10.102.1-microsoft-standard-WSL2-x86_64-with-glibc2.35)"
      },
      "RequestBody": {
        "query": "AzureActivity",
        "timespan": null,
        "additional_workspaces": null
      },
      "StatusCode": 404,
      "ResponseHeaders": {
        "Access-Control-Allow-Origin": "*",
        "Access-Control-Expose-Headers": "Retry-After,Age,WWW-Authenticate,x-resource-identities,x-ms-status-location",
        "Connection": "keep-alive",
        "Content-Length": "248",
        "Content-Type": "application/json; charset=utf-8",
        "Date": "Wed, 17 Apr 2024 01:05:12 GMT",
        "Strict-Transport-Security": "max-age=15724800; includeSubDomains",
        "Via": "1.1 draft-oms-67c68b59bf-2zz8p",
        "X-Content-Type-Options": "nosniff"
      },
      "ResponseBody": {
        "error": {
          "message": "The requested item was not found",
          "code": "ItemNotFoundError",
          "correlationId": "d8d985e1-1b27-4943-907a-959a45c87f80",
          "innererror": {
            "code": "FailedToResolveResourceError",
            "message": "Unable to resolve resource: /bad/resource/id"
          }
        }
      }
    }
  ],
  "Variables": {}
}
